src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -47,6 +47,11 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
   isPublic: boolean;
   trsLink: string;
   public refreshMessage: string;
+  modeTooltipContent = `<b>STUB:</b> Basic metadata pulled from remote site.<br />
```

Comment:
Probably can be more specific here. 
`Stub: Basic metadata pulled from source control` 
vs
`Full: Full content synced from source control`